src/main/java/frc/robot/settings/Constants.java

```diff
@@ -368,6 +368,8 @@ public CTREConfigs() {
       driveMotorConfig.CurrentLimits.SupplyCurrentLimit = DriveConstants.DRIVE_CURRENT_LIMIT;
       driveMotorConfig.CurrentLimits.SupplyCurrentLimitEnable = true;
       driveMotorConfig.CurrentLimits.StatorCurrentLimitEnable = false;
+      driveMotorConfig.CurrentLimits.SupplyCurrentThreshold = 50;
+      driveMotorConfig.CurrentLimits.SupplyTimeThreshold = 1.4;
```

Comment:
Talked with Emilio. He has some concern about using 1.4. The breaker can deteriorate over time when we go above it's limit. We should move this back to say 0.8. That should still allow us to accelaerate. We can test on field.